src/pages/NewChatPage.tsx

```diff
@@ -230,7 +231,7 @@ function NewChatPage() {
                 if (isOptionInList) {
                     newSelectedOptions = reject(selectedOptions, (selectedOption) => selectedOption.login === option.login);
                 } else {
-                    newSelectedOptions = [...selectedOptions, {...option, isSelected: true, selected: true, reportID: option.reportID ?? '-1'}];
+                    newSelectedOptions = [...selectedOptions, {...option, isSelected: true, selected: true, reportID: option.reportID ?? CONST.EMPTY_STRING}];
```

Comment:
I think this one needs to be `CONST.DEFAULT_NUMBER_ID` as well? I think going from `-1` to an empty string would have some unintended consequences.